internal/bminventory/inventory.go

```diff
@@ -2189,6 +2189,7 @@ func (b *bareMetalInventory) updateProviderParams(params installer.V2UpdateClust
 		if err != nil {
 			return fmt.Errorf("failed setting platform usages, error is: %w", err)
 		}
+		updates["platform_type"] = params.ClusterUpdateParams.Platform.Type
```

Comment:
please move this code before line 2187 - if the SetPlatformUsages will fail - you will not update the platform type in DB and this is a problem

Reply:
Done